<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonsImgWidget</class>
 <widget class="QWidget" name="CommonsImgWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="MainGrpBox">
     <property name="title">
      <string>A File</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="closeBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLay">
        <item row="1" column="0" rowspan="4">
         <widget class="QLabel" name="ThumbLbl">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Desired filename:</string>
          </property>
          <property name="buddy">
           <cstring>UploadFilenameEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="UploadFilenameEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Source:</string>
          </property>
          <property name="buddy">
           <cstring>FileSourceEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="FileSourceEdit"/>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Author:</string>
          </property>
          <property name="buddy">
           <cstring>FileAuthorEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLineEdit" name="FileAuthorEdit"/>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Date &amp;&amp; time:</string>
          </property>
          <property name="buddy">
           <cstring>FileDtEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Description</string>
          </property>
          <property name="buddy">
           <cstring>FileDescTxtEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QTextEdit" name="FileDescTxtEdit"/>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Preview…</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Info…</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QComboBox" name="FileLicenseCbx">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>License:</string>
          </property>
          <property name="buddy">
           <cstring>FileLicenseCbx</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="GeoEdit"/>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Geo:</string>
          </property>
          <property name="buddy">
           <cstring>GeoEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="StatusLbl">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="GeoBtn">
          <property name="statusTip">
           <string>View in external map…</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="FileDtEdit"/>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>UploadFilenameEdit</tabstop>
  <tabstop>FileDescTxtEdit</tabstop>
  <tabstop>FileSourceEdit</tabstop>
  <tabstop>FileAuthorEdit</tabstop>
  <tabstop>GeoEdit</tabstop>
  <tabstop>FileLicenseCbx</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
